Marlin/src/lcd/menu/menu_info.cpp

```diff
@@ -176,13 +176,7 @@ void menu_info_board() {
   STATIC_ITEM(BOARD_NAME, true, true);                           // MyPrinterController
   STATIC_ITEM(MSG_INFO_BAUDRATE ": " STRINGIFY(BAUDRATE), true); // Baud: 250000
   STATIC_ITEM(MSG_INFO_PROTOCOL ": " PROTOCOL_VERSION, true);    // Protocol: 1.0
-  #if POWER_SUPPLY == 0
-    STATIC_ITEM(MSG_INFO_PSU ": Generic", true);
-  #elif POWER_SUPPLY == 1
-    STATIC_ITEM(MSG_INFO_PSU ": ATX", true);  // Power Supply: ATX
-  #elif POWER_SUPPLY == 2
-    STATIC_ITEM(MSG_INFO_PSU ": XBox", true); // Power Supply: XBox
-  #endif
+  STATIC_ITEM(MSG_INFO_PSU ": " PSU_NAME, true);
```

Comment:
Honestly?
For what is this information god for? If i wanted to know i 'd have a look on the PSU, not crawling thru a menu to have the info on screen.
Belongs the PSU to the "board"?
Shouldn't `MSG_INFO_PSU` then at least read as "PSU-style".
Didn't we do this PR to get rid of the specific names.

Reply:
I just merge 'em, and somebody not us wanted this. But I agree, it is not pertinent to firmware.